res/adpod/truex-pod.xml

```diff
@@ -24,7 +24,7 @@
                         <Duration>00:00:30</Duration>
                         <MediaFiles>
                             <MediaFile id="coffee-video-ad" delivery="progressive" width="3840" height="2160" type="video/mp4" bitrate="48610" scalable="true" maintainAspectRatio="true">
-                                <![CDATA[https://qa-media.truex.com/reference_apps/sample_ads/video/coffee-4k.mp4]]>
+                                <![CDATA[http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerEscapes.mp4]]>
```

Comment:
I need to change these to be truex feeds.  Noting that I had to remove the above because my device did not seem to like the 4k files.

Reply:
are 4k ads a real thing? .mp4 4k videos will choke most networks and devices
they should not exist in ads in practice right?

Reply:
I would say so.  But I suppose there's nothing stopping it from happening either.  Sounds like a client / ad tech problem to deal with though.